README.md

```diff
@@ -204,6 +204,25 @@ If you are interested with specific events, it is possible with providing extra
 channel.on(ChannelState.attached, listener);
 ```
 
+#### Use of authCallback
+
+Callback that provides either tokens, or signed token requests, in response to a request with given token params.
+
+```java
+ClientOptions options = new ClientOptions();
+    
+    options.authCallback = new Auth.TokenCallback() {
+        @Override
+            public Object getTokenRequest(Auth.TokenParams params) {
```

Comment:
I think we should mention relevant return type/interface, since it returns `TokenRequest` or `TokenDetails`

Reply:
https://github.com/ably/ably-java/pull/704/commits/4f78e8c7e16b243306c4d2aba2e615bd32c1dfce